vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/annotation/Observer.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * #%L
+ * vertx-pojo-mapper-common
+ * %%
+ * Copyright (C) 2017 Braintags GmbH
+ * %%
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * #L%
+ */
+package de.braintags.vertx.jomnigate.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import de.braintags.vertx.jomnigate.observer.IObserver;
+import de.braintags.vertx.jomnigate.observer.ObserverEventType;
+
+/**
+ * The annotation defines an {@link IObserver}, which shall be applicated to the entities of a mapper.
+ * 
+ * @author Michael Remme
+ * 
+ */
+
+@Documented
+@Inherited
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+public @interface Observer {
+
+  /**
+   * Defines the observer class, which shall be used
+   * 
+   * @return the class of the observer
+   */
+  Class<? extends IObserver> observerClass();
+
+  /**
+   * Defines the priority of the given definition. Larger is higher priority.
+   * 
+   * @return the priority, default is 1
+   */
+  int priority() default 1;
```

Comment:
IMO not, cause if one is setting 0 for an observer, this value can't be overwritten any more by another observer. With end open this would be possible